cinema_service/urls.py

```diff
@@ -1,6 +1,10 @@
+from debug_toolbar.toolbar import debug_toolbar_urls
 from django.contrib import admin
 from django.urls import path
+from django.urls.conf import include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("api/cinema/", include("cinema.urls"), name="cinema"),
+    path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
The inclusion of `debug_toolbar.urls` should be wrapped in a condition to check if `DEBUG` is `True`. This will prevent the debug toolbar from being accessible in a production environment.